app/main.py

```diff
@@ -4,4 +1,31 @@
+import requests
+import os
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+city = "Paris"
+
+
+def get_weather(api_key: str) -> None:
+    weather_paris_url = f"{BASE_URL}?key={api_key}&q={city}"
```

Comment:
The `city` variable should be passed as an argument to the `get_weather` function, not used as a global variable. It's a good practice to avoid global variables when possible and pass them as function parameters instead.